Cargo.lock

```diff
@@ -4083,9 +4083,9 @@ dependencies = [
 
 [[package]]
 name = "mio"
-version = "0.8.9"
+version = "0.8.11"
 source = "registry+https://github.com/rust-lang/crates.io-index"
-checksum = "3dce281c5e46beae905d4de1870d8b1509a9142b62eedf18b443b011ca8343d0"
+checksum = "a4a650543ca06a924e8b371db273b2756685faae30f8487da1b56505a8f78b0c"
```

Comment:
Is this package used inside tests?
Or it is accidential?

Reply:
Not sure how this got pulled in. I've rebased and removed this change!